builder/src/extensionConfig.ts

```diff
@@ -235,6 +235,13 @@ function generateConfig({
     );
   }
 
+  // Add version argument when in production so the Jupyter server
+  // does not cache the files in the static file handler.
+  let filename = '[name].[contenthash].js';
+  if (mode === 'production') {
+    filename += '?v=[contenthash]';
```

Comment:
Is this actually changing the filename on disk? Or just the requests made for this file?

Reply:
Answer: it is not changing the filename on disk. It is just changing the webpack requests.